libs/shared/form-fields/src/lib/CompanySearchController/CompanySearchController.tsx

```diff
@@ -94,6 +111,34 @@ export const CompanySearchController: FC<Props> = ({
     return options || []
   }
 
+  const getIsatNumbers = (nationalId: string) => {
```

Comment:
The function name indicates return of string array of ISAT numbers, but it only returns a single string with the main ISAT number
```suggestion
  const getIsatNumber = (nationalId: string) => {
```